packages/web-components/src/components/masthead/masthead-composite.ts

```diff
@@ -693,13 +693,21 @@ class DDSMastheadComposite extends LitElement {
     } = this;
     const authenticated = userStatus !== UNAUTHENTICATED_STATUS;
     const profileItems = authenticated ? authenticatedProfileItems : unauthenticatedProfileItems;
+    const formattedLang = language?.toLowerCase().replace(/-(.*)/, m => m.toUpperCase());
+    let platformAltUrl = platformUrl;
+    if (platformUrl && formattedLang) {
+      if (typeof platformUrl === 'object' && Object.prototype.hasOwnProperty.call(platformUrl, formattedLang)) {
+        platformAltUrl = platformUrl[formattedLang].url || platformUrl;
```

Comment:
@NSDrowned just checking, this seems like an API change? We should still support the old platform url as a string so we don't break it for adopters that are currently using it with the current API. If we detect a string, we should still support but perhaps throw a deprecation notice.

Reply:
Nevermind, took another look and realized that we are allowing both, thanks @NSDrowned !